File: app/src/main/java/com/ribic/nejc/veselica/sync/PartyJobService.java
package com.ribic.nejc.veselica.sync;


import android.app.job.JobParameters;
import android.app.job.JobService;
import android.content.Intent;
import android.util.Log;

public class PartyJobService extends JobService{
    @Override
    public boolean onStartJob(JobParameters jobParameters) {
        Log.d("Party Job Service", "service started");
        Intent nowIntent = new Intent(getApplicationContext(), PartyIntentService.class);
        getApplicationContext().startService(nowIntent);
        return true;
    }

    @Override
    public boolean onStopJob(JobParameters jobParameters) {
        return false;
    }



}

File: app/src/main/java/com/ribic/nejc/veselica/sync/PartyJobService.kt
package com.ribic.nejc.veselica.sync


import android.app.job.JobParameters
import android.app.job.JobService
import android.content.Intent
import android.util.Log

class PartyJobService : JobService() {
    override fun onStartJob(jobParameters: JobParameters): Boolean {
        Log.d("Party Job Service", "service started")
        val nowIntent = Intent(applicationContext, PartyIntentService::class.java)
        applicationContext.startService(nowIntent)
        return true
    }

    override fun onStopJob(jobParameters: JobParameters): Boolean {
        return false
    }


}
